assets/prototype/infrastructure/services/toaster/useLoadingToast.js

```diff
@@ -0,0 +1,64 @@
+import { Spinner } from '@blueprintjs/core';
+import { useCallback } from '@wordpress/element';
+import useDismissToast from './useDismissToast';
+
+const useLoadingToast = (toaster, hash) => {
+	const dismiss = useDismissToast(toaster, hash);
```

Comment:
@tn3rb ESLint was complaining:
![image](https://user-images.githubusercontent.com/1477453/69608656-93a8a880-1030-11ea-8ffb-60d24af5b847.png)
 so, I moved that in outer scope and passed `dismiss` as a dependency of `useCallback`

Reply:
oh right... I know that too, but just forgot while doing that obviously, thnx